workitem/expression_compiler.go

```diff
@@ -166,6 +166,11 @@ func isInJSONContext(exp criteria.Expression) bool {
 // you can write "a->'foo' < '5'" and it will return true for the json object { "a": 40 }.
 func (c *expressionCompiler) Literal(v *criteria.LiteralExpression) interface{} {
 	json := isInJSONContext(v)
+	if _, ok := v.Value.(string); ok {
+		if strings.Contains(v.Value.(string), "Fields->") {
```

Comment:
A thought, how about adding `FieldLiteral` that will accept `(key string, val string)` and it should return constructed value like `Fields->'KEY' = 'VAL'`?

Reply:
Hmm... Then it won't be a "literal expression" anymore.

Reply:
true that but I am not very okay with `substring "Fields->"`, just saying.

Reply:
No blocker just wanted to think about it a bit.
We can go ahead to merge.

Reply:
This part of code requires some refactoring to support search across multiple table. We can fix the semantics of Field & Literal at that time.